### PR TITLE
Fix/#76

### LINKED ISSUE #76: Put the return functionality in the start text
**Explination** 

When entering the home page, enter the calendar, there is a home button, it does not work if the text is clicked.

## ⚠️ I think the only thing is that clicking on the word start does not work only if you click on the icon or back arrow.

imagen

**Expected result** 

- [ ] Clicking on the start text it returns to start

_Originally posted by @angelik0828 in https://github.com/Azordev/did-app/issues/73#issuecomment-1243953194_

### PR DESCRIPTION
**Pull request by AdrianFagilde**

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
Seems like it have repeated changes from #80 please check

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Please use this classes to the text styles:
```css
font-size: 1.1875rem;
font-weight: bold;
color: #707070;
line-height: 1.125rem;
```

Here's the design reference:
imagen

This is how looks currently:
imagen

Please add at the "Events" page too. There is the same header and the "Eventos" text should be clickable

imagen

**Review by Israel-Laguan — APPROVED** [review]

**Comment by AdrianFagilde** [comment]
Sorry, that commit was for isseus #50.

File: src/actions/EventsActions.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { EventByPKReturnTypes } from 'src/utils/apollo.types';
+import { EventByPKReturnTypes } from './../utils/apollo.types';
```
Comment (Israel-Laguan) [inline src/actions/EventsActions.ts]: Why not src?

Comment (AdrianFagilde) [reply src/actions/EventsActions.ts]: changed it a while ago on the right pull

**Comment by Israel-Laguan** [comment]
please rebase

**Review by Israel-Laguan — APPROVED** [review]

File: src/pages/Home/Home.layout.vue [hunk 2]
```diff
@@ -75,10 +75,11 @@
...
 import { HomeHeader, HomeList } from './components';
-import { ProviderCard, EventCard } from 'src/components';
-import { Event, EventType, Provider } from 'src/utils';
-import { handleUserData } from './utils/handleUserData';
-import DIDLogo from 'src/assets/logos/didperu-dark.svg';
+import { ProviderCard, EventCard } from '../../components';
+import { Event, EventType, Provider, User } from '../../utils';
+import { handleUserData } from 'src/utils';
+import DIDLogoDark from '../../assets/logos/didperu-dark.svg';
+import DIDLogo from 'src/assets/logos/didperu.svg';
```
Comment (victor-heliomar) [inline src/pages/Home/Home.layout.vue:82]:
```suggestion
import { ProviderCard, EventCard } from 'src/components';
import { Event, EventType, Provider, User, handleUserData } from 'src/utils';
import DIDLogoDark from 'src/assets/logos/didperu-dark.svg';
import DIDLogo from 'src/assets/logos/didperu.svg';
```

File: src/pages/Event/index.vue [hunk 3]
```diff
@@ -8,6 +8,7 @@
     @unsubscribe-user-to-event="onUnsubscribeUserToEvent($event)"
     @subscribe-user-to-event="() => onSubscribeUserToEvent(member_id, eventId)"
   />
+
```
Comment (heliomar-pena) [inline src/pages/Event/index.vue]:
```suggestion
```

**Comment by Israel-Laguan** [comment]
Please rebase

File: src/pages/Event/styles.scss [hunk 4]
```diff
@@ -47,4 +47,11 @@
       aspect-ratio: 1/1;
     }
   }
+
+  &__btnBack {
+    font-size: 1.1875rem;
+    font-weight: bold;
+    color: #707070;
+    line-height: 1.125rem;
+  }
```
Comment (heliomar-pena) [inline src/pages/Event/styles.scss]: Seems like the text isn't being modified by this style. Try to select directly the element where is the text, in this case is a span with class "block":

imagen

Some like this should work:

```suggestion
  &__btnBack .block {
    font-size: 1.1875rem;
    font-weight: bold;
    color: #707070;
    line-height: 1.125rem;
  }
```

**Review by heliomar-pena — CHANGES_REQUESTED** [review]

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Pull request merged**